app/views/services/edit.html.erb

```diff
@@ -37,7 +37,9 @@
             File.join(preview_service_path(service.service_id), page.url),
             target: '_blank' %>
           </li>
+          <% unless (page.type =~ /page.(checkanswers|confirmation|exit)/) %>
```

Comment:
i think we can add a method to the `MetadataPresenter::Page` model that does this for us to save us constantly calling the same thing.

`def allow_add_page?` or something like that. could be in a separate set of PRs